shiny/_template_utils.py

```diff
@@ -257,24 +444,21 @@ def directory_prompt(
 
     app_dir = questionary.path(
         "Enter destination directory:",
-        default=build_path_string(""),
+        default=path_rel_wd(template_dir.name),
         only_directories=True,
     ).ask()
 
     if app_dir is None:
         sys.exit(1)
 
-    if app_dir == ".":
-        app_dir = build_path_string(template_dir.name)
-
     return Path(app_dir)
 
 
-def build_path_string(*path: str):
+def path_rel_wd(*path: str):
     """
-    Build a path string that is valid for the current OS
+    Path relative to the working directory, formatted for the current OS
     """
-    return os.path.join(".", *path)
+    return os.path.join(".", *(path or [""]))
```

Comment:
I don't quite follow why this change is here -- both versions seem to fail on `None` and work with `""`?

Reply:
I mean I'm not too sure either, this helper function seemed to exist to create `./` using OS-specific separators. Before my change, we were calling this function `build_path_string()` and we would call it like `build_path_string("")` in the base case. 

That seems to indicate the goal is to make `"./"` and not just `"."`. The output is used as the starting value in destination directory prompts, so this change makes sense.

As it stands in this PR, you either call `path_rel_wd()` without arguments or you give it strings of path parts that are joined together.

Reply:
Ahh I got it, so this is really about calling the function with no args:

```python
>>> path_rel_wd()
'./'
>>> build_path_string()
'.'
```